engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/JModule.java

```diff
@@ -0,0 +1,16 @@
+package org.enso.compiler.core.ir;
+
+import org.enso.compiler.core.ir.module.scope.Export;
+import org.enso.compiler.core.ir.module.scope.Import;
+import org.enso.runtime.parser.dsl.IRChild;
+import org.enso.runtime.parser.dsl.IRNode;
+import scala.collection.immutable.List;
```

Comment:
If we want to get rid of Scala, then we should rather use `java.util.List`.

Reply:
I wanted the overall changes to be minimal. But it is true that there must be a lot of changes in the existing passes anyway - for example, since all the `copy` methods now will be Java methods and not Scala methods, it will not be possible to provide named arguments in Scala code.